data/campaigns/Heir_To_The_Throne/utils/httt_utils.cfg

```diff
@@ -309,7 +308,7 @@ fire:  +10%"
         [/rule]
         [rule]      # Convert cave villages to stone villages
             old=Vu
-            new=_f, Re, Uu^Vud, Uh, Uu^Uf   # wmllint: ignore
+            new=Re, Uu^Vud, Uh, Uu^Uf   # wmllint: ignore
```

Comment:
Why remove `_f` here? Though it's a little suspicious that it was there to begin with.